spaceship/spec/spec_helper.rb

```diff
@@ -6,6 +6,7 @@
 require_relative 'du/du_stubbing'
 
 unless ENV["DEBUG"]
```

Comment:
Wait, this whole block should actually be removed from the spaceship spec helper. I don't have commit access to your fork, can you update the PR?